packages/blade/src/components/Amount/Amount.tsx

```diff
@@ -1,8 +1,9 @@
 import type { ReactElement } from 'react';
```

Comment:
Now that we're supporting so many currencies it could lead to some confusion on conversion. Lets add a note in Amount's JSDOC as well as story clarifying that the Amount component will render the value as is without performing any currency conversions.